projects/Mallard/src/AppNavigation.tsx

```diff
@@ -220,6 +221,9 @@ const MainStack = () => {
 			<Main.Screen
 				name={RouteNames.Lightbox}
 				component={LightboxScreen}
+				options={{
+					...TransitionPresets.ModalSlideFromBottomIOS,
```

Comment:
Same comment as on the other review but just double checking this works on both platforms?